bodhi/client/bindings.py

```diff
@@ -287,14 +287,17 @@ def save_override(self, nvr, duration, notes):
         """
         expiration_date = datetime.datetime.utcnow() + \
             datetime.timedelta(days=duration)
-
-        return self.send_request(
-            'overrides/', verb='POST', auth=True, data={
-                'nvr': nvr,
+        data = {'nvr': nvr,
                 'expiration_date': expiration_date,
                 'notes': notes,
                 'csrf_token': self.csrf(),
-            })
+                'duration': duration}
```

Comment:
The CLI ```--duration``` seems to be a convenience that is used to [calculate the ```expiration_date```](https://github.com/fedora-infra/bodhi/blob/2.5.0/bodhi/client/bindings.py#L288-L289) so I don't think we need to implement this server-side.